js/templates/transactionsPage.html

```diff
@@ -1,9 +1,33 @@
-<nav class="bar clrP clrBr">Tabs and things go here</nav>
-<div class="pageContent">
+<nav id="pageTabBar" class="barLg clrP clrBr">
 
-  <h1>Welcome to the transactions page!</h1>
+  <div class="flexVCent pageTabs">
+    <!--consider turning tabs into a reusable view-->
+    <div>
+      <div class="userIcon disc clrBr2 clrSh1" style="background-image: url('../imgs/defaultUser.png')"></div>
+    </div>
+    <div>
+      <h1 class="h2 txUnb txUnl"><%= ob.name %></h1>
+      <div class="txt5b">Stars(x) <%= ob.polyT('userPage.followsYou') %></div>
+    </div>
+    <div class="flexExpand">
+      <div class="flexHRight flexVCent gutterH clrT2">
+        <a class="btn tab clrBr js-tab" data-tab="Home">Purchases</a>
+        <a class="btn tab clrBr js-tab" data-tab="Store">Sales</a>
+        <a class="btn tab clrBr js-tab" data-tab="Following">Cases</a>
+      </div>
+    </div>
+  </div>
```

Comment:
The userPage and transactions templates both use the same tab structure. It might be worth breaking it off into it's own template.